Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -103,9 +125,9 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
     
     /// flag that indicates if offsets calculation has already been done or not
     private var _offsetsCalculated = false
-    
-    /// array of Highlight objects that reference the highlighted slices in the chart
-    internal var _indicesToHighlight = [Highlight]()
+
+    /// The array of currently highlighted values. This might an empty if nothing is highlighted.
+    @objc open internal(set) var highlighted = [Highlight]()
```

Comment:
A) Can we keep old name, or the new name is more swift favored? 
B) If we gonna change the public var names like this, we might need to write them down and when publishing 4.0, users could know what's thre replacement.

Reply:
The public facing name was `highlighted` which is why I changed the internal var name.